src/BrewingRecipes.h

```diff
@@ -40,6 +40,9 @@ class cBrewingRecipes
 
 	/** Returns true if the item is a bottle / potion, false if not. */
 	bool IsBottle(const cItem & a_Bottle) const;
+
+	/** Returns true if the item is the fuel, false if not. */
+	bool IsFuel(const cItem & a_Bottle) const;
```

Comment:
Changed both to `a_Item`, as there are different items passed and then checked there,